README.md

```diff
@@ -65,9 +65,10 @@ Now we're in an environment that has everything set up, and we start by first in
 
 ```bash
 DIR=$(mktemp -d /tmp/dbXXX)
-# Initialize CA and server certificates. Default directory is --certs=certs
+# Initialize CA, server, and client certificates. Default directory is --certs=certs
 ./cockroach cert create-ca
 ./cockroach cert create-node 127.0.0.1 localhost $(hostname)
+./cockroach cert create-client root
```

Comment:
line 126 below is also pretty out of date. At this point I'd consider removing that (maybe our acceptance test can serve as a drop-in replacement for getting a running cluster that does... things).